docker/Dockerfile

```diff
@@ -30,8 +33,9 @@ RUN sed -i "s/httpredir\.debian\.org/ftp.us.debian.org/g" /etc/apt/sources.list
     && make -s programs \
     && make -s install \
     && cd /opt/src \
-    && rm -rf "/opt/src/libreswan-${SWAN_VER}" \
-    && apt-get -yqq remove \
+    && rm -rf "/opt/src/libreswan-${SWAN_VER}"
+
+RUN  apt-get -yqq remove \
```

Comment:
Separating the cleanup phase into a different step makes the image larger. I would keep it in the same RUN statement.